content/blog/discover-the-power-of-data-center-monitoring-using-redfish-telemetry-and-cloud-native-tooling.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Discover the power of data center monitoring using Redfish telemetry and
+  cloud-native tooling
+date: 2023-10-05T17:03:36.464Z
+author: Naveen Gupta
+authorimage: /img/img-2142.jpg
+thumbnailimage: ""
+disable: false
+---
```

Comment:
Please add the tag for your blog post:
ilo-restful-api

This will allow to tie your post with the iLO RESTful API landing page: 
https://developer.hpe.com/platform/ilo-restful-api/home/

Reply:
Fixed